aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -103,8 +103,6 @@ module aptos_framework::block {
         previous_block_votes_bitvec: vector<u8>,
         timestamp: u64
     ) acquires BlockResource {
-        timestamp::assert_operating();
```

Comment:
just curious why this is needed previously and not now.

Reply:
It seems this has come from the Diem Framework ([here](https://github.com/move-language/move/blob/main/language/documentation/examples/diem-framework/move-packages/DPN/sources/DiemBlock.move#L68)), where `timestamp::assert_operating()` was conventionally used everywhere to assert that it's not a genesis transaction, but a normal transaction after genesis. In Aptos Framework, we don't  depend on this assertion at runtime, so its meaning is little.